models/glif_psc_double_alpha.h

```diff
@@ -44,16 +44,20 @@ Description
 +++++++++++
 
 ``glif_psc_double_alpha`` provides five generalized leaky integrate-and-fire
-(GLIF) models [1]_ with alpha-function shaped synaptic currents.
+(GLIF) models [1]_ with double alpha-function shaped synaptic currents.
 Incoming spike events induce a postsynaptic change of current modeled
-by sum of two alpha functions [2]_. The alpha function is normalized such that an event
-of weight 1.0 results in a peak current of the fast component of the alpha function to
-be 1 pA at :math:`t = tau_syn_fast`. The peak current of the slow component of the alpha
-is given as amp_slow pA, at :math:`t = tau_syn_slow`. By default, glif_psc_double_alpha
-has a single synapse that is accessible through receptor_port 1. An arbitrary number of
-synapses with different time constants can be configured by setting the desired time
-constants as tau_syn_fast array. The resulting synapses are addressed through
-receptor_port 1, 2, 3, ....
+by the sum of two alpha functions (fast and slow components) for each receptor [2]_.
+This function is normalized such that an event of weight 1.0 results in a peak current
+of the fast component of the alpha function to be 1 pA at :math:`t = tau_syn_fast`.
+The relative peak current of the slow component is given as amp_slow, at
+:math:`t = tau_syn_slow`. Namely,
```

Comment:
Somehow Github is not allowing me to make inline suggestions.

An underscore in LaTeX means subscript, so I would suggest ``:math:`t = \tau_\text{syn\_slow}` ``. Same for the alpha functions (``\text{alpha\_function}``).

Reply:
Thanks! I edited newly added equations LaTeX compatible.

It looks like the documentation of the old glif_psc.h has some equations that are not properly formatted (I assume that \mathrm is not recognized somehow). Should I work on these as well in this opportunity?
https://nest-simulator.readthedocs.io/en/stable/models/glif_psc.html

Reply:
Great! Thank you for letting me know!